packages/kbn-ftr-common-functional-ui-services/services/all.ts

```diff
@@ -6,12 +6,18 @@
  * Side Public License, v 1.
  */
 
+import { RetryService } from '@kbn/ftr-common-functional-services';
 import { RetryOnStaleProvider } from './retry_on_stale';
 import { RemoteProvider } from './remote';
 import { FindProvider } from './find';
+import { TestSubjects } from './test_subjects';
+import { BrowserProvider } from './browser';
 
 export const services = {
   retryOnStale: RetryOnStaleProvider,
   __webdriver__: RemoteProvider,
   find: FindProvider,
+  testSubjects: TestSubjects,
+  retry: RetryService,
```

Comment:
Similar to my comment here https://github.com/elastic/kibana/pull/173874#discussion_r1441534610, we should not have the retry service in that list.

Reply:
Whoops!  That was from before the other merged.  On it.